spec/features/reviewing_step_by_steps_spec.rb

```diff
@@ -34,6 +35,13 @@
     and_I_can_see_additional_comments_in_the_change_note
   end
 
+  scenario "User claims step by step for 2i review" do
+    given_there_is_a_step_by_step_that_has_been_submitted_for_2i
+    when_I_view_the_step_by_step_page
+    and_I_claim_the_step_by_step_for_2i
```

Comment:
```suggestion
    then_I_can_claim_the_step_by_step_for_2i
```

Reply:
Refactoring the `reviewing_step_by_steps_spec.rb` as suggested would make the "User claims step by step for 2i review" test inconsistent with the other two above. I'm wondering if we should leave the refactor as part of the wrap-up card otherwise we can have a chat and see how can we move forward with this.